### DISCUSSION TITLE
[ Vivado Post-Synthesis Simulation ] Simulation abnormal abort

### DISCUSSION THREAD

**Discussion by fafachen61** [bol-edu/HLS-SOC-Discussions#103] [Q&A]
各位好

我在使用Vivado上完成synthesis接著執行Post-synthesis functional simulation時，發現Vivado並沒有像Behavioral simulation一樣執行testbench的內容(執行Behavioral simulation時可以正常執行)，而是出現以下WARNING訊息後停止simulation，依照terminal給出的資訊初步判斷可能與讀取pattern異常有關，但上網搜尋後仍沒有解決的方向，想請問是否有同學同樣遭遇或是有解決這種問題的方法?

以下附上tcl terminal之截圖
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/71396110/26c588b6-d39f-4c35-af4b-2d3b5da880d7)

謝謝

**Comment by s095339** [comment]
我不太確定是甚麼問題 不過我跑的時候也一開始不能跑
tb裡面是不是要load兩個dat檔案嗎 你把它改成絕對路徑試試看
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78518884/c1ad0248-7427-4c51-9a90-a6e064f82c7e)

**Comment by ZheChen-Bill** [comment]
因為原始的testbench內讀取檔案的位址是相對路徑，在vivado裡面post-synthesis functional simulation的路徑和behavior simulation的路徑兩者不同，另外說明這次作業我們並沒有要求同學繳交post-synthesis functional simulation log。
